lookout/style/format/rules.py

```diff
@@ -543,3 +544,39 @@ def __getstate__(self):
         del state["_base_model"]
         del state["_compiled"]
         return state
+
+
+class RulesEstimator(BaseEstimator, ClassifierMixin):
+
+    def __init__(self, base_model: Union[DecisionTreeClassifier, RandomForestClassifier],
+                 prune_branches: bool = True, prune_branches_algorithm: str = "top-down-greedy",
```

Comment:
We do not have to specify the type if the default value is not None: bool-s, TopDownGreedyBudget, etc.